### DISCUSSION TITLE
Vercel build failing: Error: ENOENT: no such file or directory, open '/vercel/path0/.next/server/app/index.rsc'

### DISCUSSION THREAD

**Discussion by chahore** [vercel/community#6981] [Help]
### Summary

`
○  (Static)   prerendered as static content
ƒ  (Dynamic)  server-rendered on demand
Traced Next.js server files in: 36.218ms
Created all serverless functions in: 1.833s
Collected static files (public/, static/, .next/static): 3.248ms
Error: ENOENT: no such file or directory, open '/vercel/path0/.next/server/app/index.rsc'

This is my error I am getting deploying to Vercel. I am on Next 15-rc and locally it builds and startings without issues.

### Steps to Reproduce

Here is my repo if you would like to try and running it on Vercel:
https://github.com/davidczachor/czachor-ca

**Comment by versecafe** [comment]
@davidczachor I'm not sure what's been causing this but you may want to try redeploying a few times and clearing out caches, I've had the same issue with the rc since it isn't exactly stable but it seems to have cleared up on it's own, also are you using any experimental features?

**Comment by Hachi-R** [comment]
@davidczachor did you fix this? im facing the same problem here

**Comment by yekta** [comment]
I'm having the same problem as well. Seems like it might somehow be caused by forcing static rendering. If I remove `export const dynamic = 'force-static'` on the problematic page component it's gone.

**Reply by versecafe** [reply]
It can occur even without that so I'm not quite sure what the pattern is, as I've had is self resolve even without code changes, it could be broken caches such as the `.next` or `.turbo` dirs

**Comment by laamdev** [comment]
I'm having the same exact problem, haven't found a way to make the build work in the Vercel deployment.

**Comment by laurenmacam** [comment]
same error here, works when removing experimental ppr

**Reply by rishavroy97** [reply]
correct, works after removing experimental ppr

**Reply by l-ollz** [reply]
Thanks.
But I don't know the root cause 😕

**Reply by PenumbraPL** [reply]
https://github.com/vercel/next-learn/issues/770

**Reply by eddiehe99** [reply]
It works. But how could I solve this problem if I want to use ppr?

**Reply by manutdkid77** [reply]
> It works. But how could I solve this problem if I want to use ppr?

It works if you switch to `next@canary`, run `npm install next@canary`.
[Source](https://nextjs.org/docs/app/api-reference/next-config-js/partial-prerendering)

**Reply by mickdewald** [reply]
> same error here, works when removing experimental ppr

Can confirm it works without experimental.ppr on 15-rc0 and Vercel deployment, thanks!
That's the price for riding the bleeding edge.

**Reply by amadodiazjr** [reply]
Thanks @mickdewald - removing `experimental` is what fixed this for me.

(I added this going through [the tutorial](https://nextjs.org/learn/dashboard-app/partial-prerendering))

```diff --git a/app/dashboard/layout.tsx b/app/dashboard/layout.tsx
index 84870fd..79bd4c3 100644
--- a/app/dashboard/layout.tsx
+++ b/app/dashboard/layout.tsx
@@ -1,7 +1,5 @@
 import SideNav from '@/app/ui/dashboard/sidenav';
  
-export const experimental_ppr = true;
-
 export default function Layout({ children }: { children: React.ReactNode }) {
   return (
     <div className="flex h-screen flex-col md:flex-row md:overflow-hidden">
diff --git a/next.config.mjs b/next.config.mjs
index e0ada65..81aa05b 100644
--- a/next.config.mjs
+++ b/next.config.mjs
@@ -1,9 +1,5 @@
 /** @type {import('next').NextConfig} */
 
-const nextConfig = {
-  experimental: {
-    ppr: 'incremental',
-  },  
-};
+const nextConfig = {};
 
 export default nextConfig;
```

[a comment by an automated account is not shown]
